service/src/main/java/uk/gov/hmcts/reform/fpl/enums/DocmosisTemplates.java

```diff
@@ -14,7 +14,8 @@
     ORDER_V2("FL-PLW-GOR-ENG-00728.doc", "order"),
     EPO("FL-PLW-GOR-ENG-00228.doc", "emergency_protection_order"),
     EPO_V2("FL-PLW-GOR-ENG-00744.doc", "emergency_protection_order"),
-    COVER_DOCS("FL-PLW-LET-ENG-00232.doc", "cover_documents");
+    COVER_DOCS("FL-PLW-LET-ENG-00232.doc", "cover_documents"),
+    TRANSLATION_REQUEST("WELSH.doc", "translation_request");
```

Comment:
I'd suggest to upload the form to sharepoint to have the document id (so can be reflected in local and rdo-docmosis for non prod environments - if not uploaded, the tests are going to fail in the pipeline)